examples/unixd

```diff
@@ -60,6 +60,12 @@ pam_allowed_login_groups = ["posix_group"]
 
 # home_attr = "uuid"
 
+# The prefix prepended to where home directories will be mounted to. If unset, defaults to the value
```

Comment:
minor, but could we worth expanding here what this does with an example like your use case you have to help people "see" what it does.

Reply:
I wrote some stuff, let me know if it makes sense.